component/class/defaults.yml

```diff
@@ -52,7 +52,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.83.1
+        tag: keycloak/config
```

Comment:
This will be fixed upon release of the new version of `appcat`

Reply:
You also have to update the main.yaml from the package (reminder)